addons/stock_account/models/stock_move_line.py

```diff
@@ -72,3 +72,26 @@ def _create_correction_svl(self, move, diff):
             stock_valuation_layers |= move._create_dropshipped_returned_svl(forced_quantity=abs(diff))
 
         stock_valuation_layers._validate_accounting_entries()
+
+    @api.model
+    def _should_exclude_for_valuation(self):
+        """
+        Determines if this move line should be excluded from valuation based on its ownership.
+        :return: True if the move line's owner is different from the company's partner (indicating
+                it should be excluded from valuation), False otherwise.
+        """
+        self.ensure_one()
+        if self.owner_id and self.owner_id != self.company_id.partner_id:
+            return True
+        return False
+    
+    @api.model
+    def _should_exclude_from_missing_qty_calculation(self):
+        """
+        Determines if this move line should be excluded from the calculation of the missing
+        quantity for valuation purposes based on its ownership.
+        """
+        self.ensure_one()
+        if not self.owner_id or self.owner_id == self.company_id.partner_id:
```

Comment:
BTW @AungKoKoLin1997 , this condition is the same as the one in the other method, just negated. Are you extracting 2 methods here on purpose, to be able to override each of them separately? If not, may I suggest using a single method instead? :)

Reply:
@pfertyk I'm not quite sure I understand your point. Are you referring to` _should_exclude_for_valuation()` as the other method? I believe they do not share the same condition.

Reply:
@pfertyk You're right. Thanks for suggestion. I updated the code.